.github/workflows/ci.yaml

```diff
@@ -75,6 +75,7 @@ jobs:
       # Don't remove it! We shouldn't allow CI to pass if there are any changes not committed after running gen/lint/test
       - name: Fail on dirty
         run: |
+          git diff
           git status --short
           git diff --quiet
```

Comment:
Might be worth writing an error message and/or providing the command to run to tidy up and commit changes locally, when this step fails.

Reply:
yup, good idea